book/http.md

```diff
@@ -857,55 +833,36 @@ make it so that, if your browser is started without a URL being given,
 some specific file on your computer is opened. You can use that file
 for quick testing.
 
+*Alternate encodings:* read the encoding from the `Content-Type`
+header instead of always using `utf8`. Test it on a real site that
+doesn't use `utf8`, like `google.com`.
+
 *data:* Yet another scheme is *data*, which
 allows inlining HTML content into the URL itself. Try navigating to
 `data:text/html,Hello world!` in a real browser to see what happens. Add
 support for this scheme to your browser. The *data* scheme is especially
 convenient for making tests without having to put them in separate files.
 
-*Body tag:* Only show text in an HTML document if it is between
```

Comment:
Too easy?

Reply:
Students would solve it by splitting on `<body>`; there's a comment about not doing that but it feels poorly motivated. But more importantly if you implement this it then gets in the way at a later point, probably Ch 6 when you find you don't have any `<link>` elements for some reason.